tools/fasta_compute_length/fasta_compute_length.xml

```diff
@@ -1,8 +1,11 @@
-<tool id="fasta_compute_length" name="Compute sequence length" version="1.0.3" profile="16.04">
+<tool id="fasta_compute_length" name="Compute sequence length" version="1.0.3+galaxy1" profile="22.04">
```

Comment:
Maybe a patch version number bump makes more sense, as there isn't an underlying tool version to align to?

```suggestion
<tool id="fasta_compute_length" name="Compute sequence length" version="1.0.4" profile="22.04">
```